e2e-tests/tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
+
+[testenv]
+passenv = PYTEST_ADDOPTS PYTEST_BASE_URL SAUCELABS_API_KEY SAUCELABS_USERNAME
+deps =
+    bidpom==2.0.1
+    PyPOM==1.1.1
+    pytest==3.0.5
+    pytest-base-url==1.2.0
+    pytest-html==1.13.0
+    pytest-selenium==1.7.0
+    pytest-variables==1.4
+    pytest-xdist==1.15.0
+    selenium==3.0.2
+    requests==2.12.5
```

Comment:
Agreed; just leaving this as-is, for now, but we can certainly revisit this in the future, Conan.